cfme/tests/cloud/test_providers.py

```diff
@@ -186,6 +189,27 @@ def vm_ip(cfme_vhd, pwsh_ssh):
         pwsh_ssh.run_command("pwsh {}".format(path_script), timeout=180)
 
 
+@pytest.fixture
+def ec2_provider_with_sts_creds(appliance):
+    assume_role_creds = cfme_data['management_systems']['ec2west']['sts_assume_role']['credentials']
+    creds = Credential(principal=credentials[assume_role_creds]['username'],
+                       secret=credentials[assume_role_creds]['password'])
+    endpoint = EC2Endpoint(
+        assume_role_arn=cfme_data['management_systems']['ec2west']['sts_assume_role']['role_arn'],
```

Comment:
Same here, should be accessed through `provider.data`

Reply:
Reworked